<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>基本介绍</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" rowspan="2" colspan="2">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>190</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>打开</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>新建</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item row="0" column="3">
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item row="0" column="4">
         <widget class="QTimeEdit" name="timeEdit"/>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="nameline">
          <property name="text">
           <string>庄周</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>逍遥游</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QListWidget" name="listWidget">
          <item>
           <property name="text">
            <string>且夫水之积也不厚</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>则其负大舟也无力</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>覆杯水于坳堂之上</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>则芥为之舟</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>置杯焉则胶</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>水浅而舟大也</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>风之积也不厚</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>则其负大翼也无力</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>故九万里后</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>则风斯在下矣</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>而后乃今培风</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>背负青天</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>而莫之夭阏者</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="3" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>北冥有鱼嘛？</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>114</width>
             <height>101</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="yesRadioButton">
              <property name="text">
               <string>有</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="noRadioButton">
              <property name="text">
               <string>没有</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>40</y>
             <width>94</width>
             <height>136</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkBoxFish">
              <property name="text">
               <string>大鱼</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxBird">
              <property name="text">
               <string>鹏</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxKun">
              <property name="text">
               <string>鲲</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxBoth">
              <property name="text">
               <string>以上都是</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>能力设定</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="labelAttri">
          <property name="text">
           <string>属性</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelSkill">
          <property name="text">
           <string>出装推荐</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableAttri"/>
        </item>
        <item row="1" column="1">
         <widget class="QTableWidget" name="tableSkill"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>扩展信息</string>
       </attribute>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>581</width>
          <height>431</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>日历效果</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCalendarWidget" name="calendarWidget"/>
        </item>
        <item>
         <widget class="QCalendarWidget" name="calendarWidget_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1175</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="separator"/>
    <addaction name="openAction"/>
    <addaction name="separator"/>
    <addaction name="saveAction"/>
    <addaction name="separator"/>
    <addaction name="deleteAction"/>
    <addaction name="closeAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>选项(O)</string>
    </property>
    <widget class="QMenu" name="rejuAction">
     <property name="title">
      <string>焕肤(R)</string>
     </property>
     <addaction name="styleWhite"/>
     <addaction name="separator"/>
     <addaction name="styleBlack"/>
    </widget>
    <addaction name="fontAction"/>
    <addaction name="separator"/>
    <addaction name="colorAction"/>
    <addaction name="separator"/>
    <addaction name="rejuAction"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="helpAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_H"/>
  </widget>
  <action name="newAction">
   <property name="text">
    <string>新建文件(N)</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>打开文件(O)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>保存文件(S)</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>删除文件(D)</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="icon">
    <iconset>
     <normaloff>:/image/msg_question.png</normaloff>
     <normalon>:/image/msg_question.png</normalon>:/image/msg_question.png</iconset>
   </property>
   <property name="text">
    <string>帮助文档(T)</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于(A)</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="text">
    <string>字体(F)</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="text">
    <string>颜色(C)</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>关闭退出(C)</string>
   </property>
  </action>
  <action name="styleWhite">
   <property name="text">
    <string>白色靓丽</string>
   </property>
  </action>
  <action name="styleBlack">
   <property name="text">
    <string>黑色炫酷</string>
   </property>
   <property name="iconText">
    <string>黑色炫酷</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
